br/pkg/task/stream.go

```diff
@@ -1412,6 +1412,10 @@ func restoreStream(
 	downstreamIdset := make(map[int64]struct{})
 	for upstreamId, rule := range rewriteRules {
 		downstreamId := restoreutils.GetRewriteTableID(upstreamId, rule)
+		info, ok := mgr.GetDomain().InfoSchema().TableInfoByID(downstreamId)
```

Comment:
After a few versions, domain will use infoschemaV2 in default. So it may get some tables' info from TiKV instead of memory cache.
Therefore, it can check whether this restore is only has log restore data and if not it can skip check tiflash replica. In most cases, the restore will have a full backup data, and no need to check tiflash replica.

Reply:
When we check tiflash replica here, we already restore the metakvs into the cluster. So if it returns the error to the user, the cluster has fall into chaotic situation.
Besides, we can ensure no metakvs table with tiflash replica is restored during log restore (see Line:1381 in br/pkg/task/stream.go). Maybe we can check it before restore metakvs?

Reply:
ok